packages/api/src/services/collateral-prices.ts

```diff
@@ -13,6 +13,18 @@ type Dependencies = {
   appClients: AppClients;
 };
 
+const tokenAddressesMap = {
+  uma: {
```

Comment:
this seems unecessary, its basically ignored by the logic

Reply:
I liked this approach more than leaving comments about how to group the addresses